kaolin/graphics/SoftRenderer.py

```diff
@@ -73,14 +89,14 @@ def __init__(
             viewing_angle: float = 30.,
             viewing_scale: float = 1.0, 
             eye: torch.Tensor = None,
-            camera_direction: torch.Tensor = torch.FloatTensor([0, 0, 1]),
-            near: float = 0.1, far: float = 100,
+            camera_direction: torch.Tensor = None,
+            near: float = 1, far: float = 100,
```

Comment:
there is an inconsistent style (2 arguments per line)
Also the default value is not consistent with the documentation